types/src/transaction/deploy.rs

```diff
@@ -482,6 +482,22 @@ impl Deploy {
             });
         }
 
+        let wasm_lane_limit = config
+            .transaction_v1_config
+            .get_max_transaction_gas_limit(LARGE_WASM_LANE_ID);
+        let wasm_lane_limit_as_gas = Gas::new(wasm_lane_limit);
```

Comment:
aren't detection of lanes & sizes supposed to be determined at runtime, not hardcoded?

Reply:
Switched to make it be determined runtime